polyfill/lib/absolute.mjs

```diff
@@ -97,13 +97,12 @@ export class Absolute {
     const twons = GetSlot(two, EPOCHNANOSECONDS);
     const diff = twons.minus(onens);
 
-    const ns = diff.mod(1e3);
-    const us = diff.divide(1e3).mod(1e3);
-    const ms = diff.divide(1e6).mod(1e3);
-    const ss = diff.divide(1e9);
-
-    const Duration = ES.GetIntrinsic('%Temporal.Duration%');
-    const duration = new Duration(0, 0, 0, 0, 0, ss, ms, us, ns, 'balance');
+    const duration = ES.ToDuration({
```

Comment:
Can author code mess with this by setting `Object.prototype.minutes`? `Object.create(null, {..})` would avoid that, I think. Test would be good as well.

Reply:
I ended up writing this a different way after rebasing it this morning, but this is still something to test for, I think.

Reply:
My conclusion is that while `Object.prototype.minutes = 5489755` will badly mess with `Duration.from({...})` (and Duration.with if we add it), it will not affect this code the way it's currently written. I'm not sure what approach we should generally take in adding tests for this sort of thing to the test suite.